<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 1px solid;
border-color: rgb(148, 168, 199);
border-radius: 10px;
background: white;
}

QPushButton:hover{
border-color: rgb(0, 0, 199);
}

QPushButton:pressed{
background: rgb(219, 225, 233);
}</string>
    </property>
    <property name="text">
     <string>Контакты</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background:rgba(192, 199, 204,0.4);
border: 2px solid;
border-color:rgb(148, 168, 199);
border-radius: 10px;
}
</string>
    </property>
    <widget class="QLabel" name="labelRemoteName">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color:rgba(17, 191, 75, 1);
border:none;
background-color:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/img/img/ring5.png);
border:none;
}

QPushButton:hover{
border-image:url(:/new/img/img/ring4.png);
border:none;
}

QPushButton:pressed{
border-image:url(:/new/img/img/ring6.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/img/img/ring1.png);
border:none;
}

QPushButton:hover{
border-image: url(:/new/img/img/ring2.png);
border:none;
}

QPushButton:pressed{
border-image: url(:/new/img/img/ring3.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/img/img/ring1.png);
border:none;
}

QPushButton:hover{
border-image: url(:/new/img/img/ring2.png);
border:none;
}

QPushButton:pressed{
border-image: url(:/new/img/img/ring3.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/img/img/ring7.png);
border:none;
}

QPushButton:hover{
border-image:url(:/new/img/img/ring8.png);
border:none;
}

QPushButton:pressed{
border-image: url(:/new/img/img/ring9.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-image: url(:/new/img/img/ring10.png);
border:none;
}

QPushButton:hover{
border-image: url(:/new/img/img/ring11.png);
border:none;
}

QPushButton:pressed{
border-image: url(:/new/img/img/ring12.png);
border:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border: 1px solid;
border-color: rgb(148, 168, 199);
border-radius: 10px;
background: white;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border:none;
background:none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpenDialog"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenDialog">
   <property name="text">
    <string>Диалоговое окно</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
